--- ./lib/stdint.h~	2012-01-05 20:30:43.531378887 +1030
+++ ./lib/stdint.h	2012-01-05 20:32:57.307382370 +1030
@@ -504,10 +504,10 @@
   /* BSD/OS 4.0.1 has a bug: <stddef.h>, <stdio.h> and <time.h> must be
      included before <wchar.h>.  */
 # include <stddef.h>
-# include <stdio.h>
-# include <time.h>
+//# include <stdio.h>
+//# include <time.h>
 # define _GL_JUST_INCLUDE_SYSTEM_WCHAR_H
-# include <wchar.h>
+//# include <wchar.h>
 # undef _GL_JUST_INCLUDE_SYSTEM_WCHAR_H
 #endif
 #undef WCHAR_MIN
